crates/pathfinder/src/bin/pathfinder/config.rs

```diff
@@ -91,6 +91,14 @@ Examples:
         env = "PATHFINDER_SQLITE_WAL", 
     )]
     sqlite_wal: bool,
+
+    #[arg(
+        long = "max-connections",
```

Comment:
Wouldn't `max-rpc-connections` (as Mirko suggested) be a better name here?

Reply:
Ah yes, I somehow thought that was in 😅

`max-connections` will almost definitely conflict with other p2p or similar things in the future.

Reply:
Missed the rename. Fixing now.

Reply:
Done